### PR TITLE
Parsing json results of cube service

### PR DESCRIPTION
**Pull request by edgan8**
Parsing the json outputs of the cube service response into dataframes.

aria_api_cube contains generic data in the same shape as we will expect from the cube service

### CODE REVIEW

File: core/src/main/java/edu/stanford/futuredata/macrobase/contrib/aria/APICubeResult.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package edu.stanford.futuredata.macrobase.contrib.aria;
```
Comment (pbailis) [inline core/src/main/java/edu/stanford/futuredata/macrobase/contrib/aria/APICubeResult.java]: kind of strange this lives in `core` even though it's really `contrib`... do we want to rethink here, or just let core contain basically everything

File: core/src/main/java/edu/stanford/futuredata/macrobase/contrib/aria/APICubeResult.java [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+import java.util.List;
+
+public class APICubeResult {
+    public String interval;
+    public String granularity;
```
Comment (pbailis) [inline core/src/main/java/edu/stanford/futuredata/macrobase/contrib/aria/APICubeResult.java]: comments here to describe what each field means would be really useful; e.g., what is interval vs granularity?

File: core/src/main/java/edu/stanford/futuredata/macrobase/contrib/aria/APIFilterResult.java [hunk 3]
```diff
@@ -0,0 +1,13 @@
+package edu.stanford.futuredata.macrobase.contrib.aria;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+import java.util.List;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class APIFilterResult {
+    public String dimension;
+    public String operation;
```
Comment (pbailis) [inline core/src/main/java/edu/stanford/futuredata/macrobase/contrib/aria/APIFilterResult.java]: Ditto, comments here would be really helpful.

File: core/src/main/java/edu/stanford/futuredata/macrobase/contrib/aria/APISeriesResult.java [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+import java.util.List;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class APISeriesResult {
+    public String operation;
+    public List<Double> values;
+    public String segment;
```
Comment (pbailis) [inline core/src/main/java/edu/stanford/futuredata/macrobase/contrib/aria/APISeriesResult.java]: Ditto, comment, e.g., what is 'segment'?

**Comment by edgan8** [comment]
No longer needed since we have removed most cube specific code!

**Pull request closed without merging**